packages/generator-clay-component/app/templates/src/tests/_Boilerplate.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * © 2018 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import * as React from 'react';
+import * as TestRenderer from 'react-test-renderer';
+import <%= componentName %> from '../<%= componentName %>';
+
+describe('<%= componentName %>', function() {
+	it('should render', () => {
```

Comment:
Style: generally "should" should be avoided in example descriptions. You can always express it more concisely by leading directly with a verb. This is useful because it makes it more likely that you can fit the description in a single line. So in this case it would be:

```suggestion
	it('renders', () => {
```